elements/drag-drop-elements/pl-drag-drop-element.css

```diff
@@ -0,0 +1,79 @@
+.connectedSortable {
+  /* applies to MCQ options draggable blocks*/
+  border: 3px solid #eee;
+  border-radius: 4px;
+  /*width: 302px;*/
+  background-color: #efefff;
+  min-width: 200px;
+  width: 350px;
```

Comment:
I don't think its wise to hardcode the width here to a particular number of pixels. Maybe a better way is to have the width declared as a percentage in the `.mustache` file, and a config option can define the width in order to make the panels either side by side or on top of one another?

Reply:
yep, the spacing was something I figured at the very beginning when I started making the element, it's an arbitrary magic number that I thought looked ok.

ill make this an option as you suggested?

Reply:
People seemed excited about having it as an option at last meeting, so ya, go for it.

Reply:
added a layout setting in latest commit